### PR TITLE
[Bug]Fix ventus memfill

### PR DESCRIPTION
**Pull request by summersurface**
Implementation of ventus memfill interface.

### CODE REVIEW

**Review by zhoujingya — COMMENTED** [review]
Remove???

**Comment by zhoujingya** [comment]
刚看了一下CTS的一些测试，你有测试这个功能实现之后，测试有没有过，比如`test_conformance/api/test_zero_sized_enqueue.cpp`这个测试

File: lib/CL/devices/ventus/pocl_ventus.cc [hunk 1]
```diff
@@ -1312,6 +1311,20 @@ pocl_ventus_copy_rect (void *data, pocl_mem_identifier *dst_mem_id,
     free(dst_buff);
 }
 
+void
+pocl_ventus_memfill (void *data, pocl_mem_identifier *dst_mem_id,
+                     cl_mem dst_buf, size_t size, size_t offset,
+                     const void *__restrict__ pattern, size_t pattern_size)
+{
+  struct vt_device_data_t *d = (struct vt_device_data_t *)data;
+  void *host_ptr = pocl_aligned_malloc(MAX_EXTENDED_ALIGNMENT, size);
```
Comment (Jules-Kong) [inline lib/CL/devices/ventus/pocl_ventus.cc:1320]: Why can't it be something like "char *buff = (char *)malloc(size);"? What is the principle of alignment here?

File: lib/CL/devices/ventus/pocl_ventus.cc [hunk 2]
```diff
@@ -501,8 +502,6 @@ step5 make a writefile for chisel
               else
                 {
                   cl_mem m = (*(cl_mem *)(al->value));
-                  err=pocl_ventus_alloc_mem_obj(cmd->device, m, m->mem_host_ptr);
```
Comment (Jules-Kong) [inline lib/CL/devices/ventus/pocl_ventus.cc:504]: What is the testcase where this error occurs?

**Comment by Jules-Kong** [comment]
Combine all commits into one.

**Review by Jules-Kong — APPROVED** [review]
lgtm

**Pull request merged**